lms/models/__init__.py

```diff
@@ -8,7 +8,14 @@
 from lms.models.file import File
 from lms.models.grading_info import GradingInfo
 from lms.models.group_info import GroupInfo
-from lms.models.grouping import CanvasGroup, CanvasSection, Course, Grouping
+from lms.models.grouping import (
+    BlackboardGroup,
+    CanvasGroup,
+    CanvasSection,
+    Course,
+    Grouping,
+    GroupingMembership,
```

Comment:
Oh I think I merged a PR that added this same line (GroupingMembership) before seeing this, this will need a rebase.